src/azure.js

```diff
@@ -3,28 +3,36 @@ var api         = require('./v1');
 
 api.declare({
   method:     'get',
-  route:      '/azure/:account/table/:table/read-write',
+  route:      '/azure/:account/table/:table/:level',
   name:       'azureTableSAS',
   input:      undefined,
   output:     'azure-table-access-response.json#',
   deferAuth:  true,
   stability:  'stable',
-  scopes:     [['auth:azure-table-access:<account>/<table>']],
+  scopes:     [['auth:azure-table-access:<account>/<table>'], ['auth:azure-table-access:<account>/<table>/<level>']],
   title:      "Get Shared-Access-Signature for Azure Table",
   description: [
     "Get a shared access signature (SAS) string for use with a specific Azure",
-    "Table Storage table.  Note, this will create the table, if it doesn't",
-    "already exist."
+    "Table Storage table. By not specifying a level as in azureTableSASLevel,",
+    "you will get read-write permissions. If you get read-write from this, it will create the",
+    "table if it doesn't already exist.",
   ].join('\n')
 }, async function(req, res) {
-  // Get parameters
   var account   = req.params.account;
   var tableName = req.params.table;
+  var level     = req.params.level || 'read-write';
```

Comment:
No need for the `|| 'read-write'`, the old clients have it embedded in their path.